astrogrism/HST/transform_models.py

```diff
@@ -40,19 +43,20 @@ class WFC3IRForwardGrismDispersion(Model):
     n_inputs = 5
     n_outputs = 4
 
-    def __init__(self, orders, lmodels=None, xmodels=None,
-                 ymodels=None, theta=0., name=None, meta=None):
+    def __init__(self, orders, lmodels=None, xmodels=None, ymodels=None,
+                 theta=0., name=None, meta=None, l_unit="micron"):
```

Comment:
Food for thought, why don't we just construct the unit as the default value here (i.e. `l_unit=u.micron`)? Seems more robust to me, since we already have to convert to astropy units to use it, right?

Reply:
Good suggestion, next time one of us makes a PR we can try to remember. There are definitely some other little code improvements throughout to be made at some point.